src/docfx/lib/log/Error.cs

```diff
@@ -56,10 +60,14 @@ public Error WithCustomRule(CustomRule customRule, bool? enabled = null)
                 level = ErrorLevel.Off;
             }
 
+            var message = string.IsNullOrEmpty(customRule.OverrideMessage) ?
+                string.IsNullOrEmpty(customRule.AdditionalMessage) ? Message : $"{Message}{(Message.EndsWith('.') ? "" : ".")} {customRule.AdditionalMessage}"
+                : string.Format(customRule.OverrideMessage, OverrideParameters);
```

Comment:
Replicate the exception handling here:
https://dev.azure.com/ceapex/Engineering/_git/Docs.Validation?path=%2FValidation%2FAbstractValidator.cs&version=GBmaster&line=138&lineEnd=138&lineStartColumn=13&lineEndColumn=18&lineStyle=plain

Reply:
OK.